test/unit/lib/getPharmacies.js

```diff
@@ -13,18 +13,18 @@ const expect = chai.expect;
 describe('Nearby', () => {
   let geo = [];
   const searchPoint = { latitude: 53.797431921096, longitude: -1.55275457242333 };
-  const sillySearchPoint = { latitude: 49.9126167297363, longitude: -6.30890274047852 };
+  const remoteSearchPoint = { latitude: 49.9126167297363, longitude: -6.30890274047852 };
 
   describe('happy path', () => {
-    beforeEach('load data', () => {
+    before('load data', () => {
```

Comment:
Yeah, its gone as it wasn't doing anything useful. I *think* it was necessary at some point and when its been changed it just hasn't been updated.